templates/taxi/driver_list.html

```diff
@@ -5,10 +5,17 @@ <h1>Driver list</h1>
   {% if driver_list %}
     <ul>
       {% for driver in driver_list%}
-        <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
-        </li>
+        {% if driver.username == user.username %}
+          <li>
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+            ({{ driver.first_name }} {{ driver.last_name }}) (Me)
+          </li>
+        {% else%}
+          <li>
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+            ({{ driver.first_name }} {{ driver.last_name }})
+          </li>
+        {% endif %}
```

Comment:
Code style issue: There is no space before the `%}` at the end of the lines. This is a minor issue, but it's a good practice to keep your code consistent and follow the Django template style guide which recommends leaving spaces around template tags.